### PR TITLE
Fix dotnet pack with build base path

### PR DESCRIPTION
**Pull request by pakrym**
https://github.com/dotnet/cli/issues/1525

/cc @livarcocc

### CODE REVIEW

File: test/Microsoft.DotNet.Tools.Tests.Utilities/Commands/PackCommand.cs [hunk 1]
```diff
@@ -23,6 +24,15 @@ private string OutputOption
                                            $"-o \"{_outputDirectory}\"";
             }
         }
+        private string BuildBasePathOption
+        {
+            get
+            {
+                return _buildBasePath == string.Empty ?
```
Comment (livarcocc) [inline test/Microsoft.DotNet.Tools.Tests.Utilities/Commands/PackCommand.cs]: string.IsNullOrEmpty()?

**Comment by livarcocc** [comment]
LGTM

File: test/Microsoft.DotNet.Tools.Tests.Utilities/Commands/PackCommand.cs (hunk 1 above)
Comment (pakrym) [inline test/Microsoft.DotNet.Tools.Tests.Utilities/Commands/PackCommand.cs]: `buildBasePath == string.Empty ` is common pattern for Command class options see https://github.com/dotnet/cli/pull/1535/files#diff-adee0a3160968bab49c8c4835c82cc86R61 https://github.com/dotnet/cli/pull/1535/files#diff-adee0a3160968bab49c8c4835c82cc86R51 and much more

**Comment by pakrym** [comment]
@dotnet-bot test Windows_NT Debug Build please
@dotnet-bot test Windows_NT Release Build please

**Pull request closed without merging**